lib/src/lifecycle_module.dart

```diff
@@ -264,18 +247,100 @@ abstract class LifecycleModule extends SimpleModule
   /// Whether the module is currently unloading.
   bool get isUnloading => _state == LifecycleState.unloading;
 
-  @override
-  StreamSubscription<T> listenToStream<T>(
-          Stream<T> stream, void onData(T event),
-          {Function onError, void onDone(), bool cancelOnError}) =>
-      _disposableProxy.listenToStream(stream, onData,
-          onError: onError, onDone: onDone, cancelOnError: cancelOnError);
-
   //--------------------------------------------------------
   // Public methods that can be used directly to trigger
   // module lifecycle / check current lifecycle state
   //--------------------------------------------------------
 
+  /// Public method to dispose of this module.
```

Comment:
🌵 This statement is more-or-less a rehash of the function signature. Maybe this would be more apt for a doc comment:

```diff
-  /// Public method to dispose of this module.
+  /// Disposes this module and all its disposable dependencies.
```